data/remnant ships.txt

```diff
@@ -317,30 +469,101 @@ ship "Gascraft"
 		"shields" 800
 		"hull" 1500
 		"required crew" 1
-		"bunks" 1
-		"mass" 80
+		"bunks" 2
+		"mass" 78
 		"drag" 1.7
 		"heat dissipation" 0.9
 		"fuel capacity" 700
-		"cargo space" 8
-		"outfit space" 90
+		"energy capacity" 150
+		"ramscoop" 0.5
+		"cargo space" 9
+		"outfit space" 88
 		"engine capacity" 34
 		"shield generation" 0.6
 		"shield energy" 0.4
 		"hull repair rate" 1.2
 		"hull energy" 0.9
 		"gaslining" 1
+		"outfit scan power" 50
+		"outfit scan speed" 1
+		"tactical scan power" 50
+		"atmosphere scan" 100
+
 	outfits
 		"Millennium Cell"
-		"Supercapacitor"
 		"Emergency Ramscoop"
 		"Quantum Key Stone"
-		
+
 		"Crucible-Class Thruster"
 		"Crucible-Class Steering"
 		"Hyperdrive"
-	
+
 	engine 0 30
 	explode "tiny explosion" 8
 	explode "small explosion" 7
 	description "Centuries ago, the Remnant built these ships in order to explore the upper atmosphere of nearby gas giants. The Archons took issue with the Remnant's research methods, and the studies were unable to be completed. The remaining gascraft have been gathering dust in storage for centuries while the Remnant waits to see if the anger of the Archons will subside."
+
+
+
+ship "Pelican"
+	sprite "ship/pelican"
+	thumbnail "thumbnail/pelican"
+	attributes
+		category "Heavy Freighter"
+		licenses
+			Remnant
+		cost 9770000
+		"shields" 11300
+		"hull" 5700
+		"required crew" 14
+		"bunks" 35
+		"cargo space" 302
+		"mass" 320
+		"drag" 7.2
+		"heat dissipation" 0.7
+		"fuel capacity" 600
+		"ramscoop" 3
+		"outfit space" 438
+		"weapon capacity" 157
+		"engine capacity" 129
+		"shield generation" 2.7
+		"shield energy" 1.8
+		"hull repair rate" 1
+		"hull energy" 0.75
+		weapon
+			"blast radius" 160
+			"shield damage" 1600
+			"hull damage" 800
+			"hit force" 2400
+	outfits
+		"Thrasher Cannon" 2
+		"Thrasher Turret"
+		"Point Defense Turret" 2
+
+		"Aeon Cell"
+		"Crystal Capacitor"
+		"Thermoelectric Cooler" 2
+		"Emergency Ramscoop"
+		"Quantum Key Stone"
+		"Scanner Software"
+		"Tuning Rifle" 15
+
+		"Forge-Class Thruster"
+		"Smelter-Class Steering"
+		"Hyperdrive"
+
+	engine 22 92 0.7
+	engine -22 92 0.7
+	engine 0 114 1.1
+	gun 8 -135
+	gun -8 -135
+	turret 0 -67 "Thrasher Turret"
+	turret 56 35 "Point Defense Turret"
+	turret -56 35 "Point Defense Turret"
+	explode "small explosion" 15
+	explode "medium explosion" 30
+	explode "large explosion" 20
+	explode "huge explosion" 5
+	"final explode" "final explosion large" 1
+	description "The Pelican is closely based on the original semi-organic ships the Remnant found in the Ember Wastes. While they are not technically alive, their internal mechanisms and method of construction bear such a close resemblance to living organisms that they are often considered as such. These ships serve as the backbone of Remnant salvage efforts, as well as their resource distribution network."
```

Comment:
"Ember Waste" should be singular. Also, they found semi-organic *materials*, not ships; that would imply that they ran into the original species, or found their old ships in archives, which I am pretty sure is not the case.

Reply:
Changed both of those. Thanks!